### PR TITLE
A lot of improvements

### PR DESCRIPTION
**Pull request by Xanders**
Please, see Changelog (version 1.0.0) for details:
https://github.com/Xanders/pry-remote-em/blob/a_lot_of_improvements/CHANGELOG

### CODE REVIEW

File: lib/pry-remote-em/broker.rb [hunk 1]
```diff
@@ -179,15 +193,16 @@ def start_tls
 
     def peer_ip
       return @peer_ip if @peer_ip
-      return "" if get_peername.nil?
+      return '' if get_peername.nil?
       @peer_port, @peer_ip = Socket.unpack_sockaddr_in(get_peername)
+      @peer_ip = '127.0.0.1' if @peer_ip == '::1' # Little hack to avoid segmentation fault in EventMachine 1.2.0.1 while connecting to PryRemoteEm Server from localhost with IPv6 address
```
Comment (gruis) [inline lib/pry-remote-em/broker.rb]: 👍

File: pry-remote-em.gemspec [hunk 2]
```diff
@@ -8,12 +8,13 @@ Gem::Specification.new do |s|
   s.homepage      = 'http://github.com/simulacre/pry-remote-em'
   s.email         = 'pry-remote-em@simulacre.org'
   s.authors       = ['Caleb Crane']
```
Comment (gruis) [inline pry-remote-em.gemspec]: don't forget to add your name @Xanders

File: pry-remote-em.gemspec [hunk 3]
```diff
@@ -8,12 +8,13 @@ Gem::Specification.new do |s|
   s.homepage      = 'http://github.com/simulacre/pry-remote-em'
```
Comment (gruis) [inline pry-remote-em.gemspec]: this should now be `'https://github.com/gruis/pry-remote-em'`

File: lib/pry-remote-em/sandbox.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
+module PryRemoteEm
+  class Sandbox
```
Comment (gruis) [inline lib/pry-remote-em/sandbox.rb]: A short description for the purpose of `Sandbox` would be helpful for sleepy people like me. :)

File: lib/pry-remote-em/proto.rb [hunk 5]
```diff
@@ -1,59 +1,20 @@
-require 'json'
-require "zlib"
+require 'msgpack'
```
[the remark this answers is not in the document]
Comment (Xanders) [reply lib/pry-remote-em/proto.rb]: Ok, I added the comment. Please, tell me if you recollect the reason not to use MessagePack.

File: pry-remote-em.gemspec (hunk 2 above)
Comment (Xanders) [reply pry-remote-em.gemspec]: Oh, thank you! :) Done!

File: pry-remote-em.gemspec (hunk 3 above)
Comment (Xanders) [reply pry-remote-em.gemspec]: Done.

File: lib/pry-remote-em/sandbox.rb (hunk 4 above)
Comment (Xanders) [reply lib/pry-remote-em/sandbox.rb]: I agree, I added a big paragraph in Readme!

**Comment by Xanders** [comment]
Thank you so much!

I verified terminal interaction on OSX and in Ubuntu Docker container, both works well. In the past I used the lib on Windows, it worked.

I'll merge it and push new gem version into RubyGems.

**Pull request merged**